### ISSUE TITLE
Firefox extension opens a little window always and takes the focus

### ISSUE THREAD

**Issue by amontero** [danny0838/webscrapbook#359]
In Firefox extension, the dialog box that appears when the extension get the page is always displayed in the foreground for a short time but long enough to take the focus of Firefox, it happens even if the option "Automatically close dialog box when capture is finished" is set to "always". Is there any way to prevent this dialog box from showing and taking Firefox's focus?

**Comment by danny0838** [comment]
No, there's a current bug of Firefox that prevents this config from taking effect.

More details: https://bugzilla.mozilla.org/show_bug.cgi?id=1213484

**Comment by danny0838** [comment]
Keep this issue open as we need to update the code when Firefox fixes the API.
